tidy3d/components/data/monitor_data.py

```diff
@@ -2232,8 +2231,10 @@ def power(self) -> xr.DataArray:
         ``xarray.DataArray``
             Power at points relative to the local origin.
         """
-        power_theta = 0.5 * np.real(self.Etheta.values * np.conj(self.Hphi.values))
-        power_phi = 0.5 * np.real(-self.Ephi.values * np.conj(self.Htheta.values))
+        conj = np.vectorize(np.conj)
+        real = np.vectorize(np.real)
+        power_theta = 0.5 * real(self.Etheta.values * conj(self.Hphi.values))
```

Comment:
do we need to do this treatment in other parts of this class? I think this kind of operation does pop up from time to time?

Reply:
If we want to differentiate w.r.t. those methods then yeah this needs to be done. Although I'm hoping that we can get rid of this "hack" with a DataArray refactor...